packages/slice-machine/components/Forms/CreateSliceModal.tsx

```diff
@@ -7,7 +7,7 @@ import camelCase from "lodash/camelCase";
 import startCase from "lodash/startCase";
 import { InputBox } from "./components/InputBox";
 import { RESERVED_SLICE_NAME } from "@lib/consts";
-import { SliceSM } from "@core/models";
+import { SliceSM } from "@lib/models/common/Slice";
```

Comment:
Linter warning on line 3.... at least we know the linter is working :)

Reply:
Thanks, this pointed out a piece of code I removed temporarily while fixing tests. The code has been restored, and the failing test was fixed.